saleor/graphql/site/mutations.py

```diff
@@ -0,0 +1,55 @@
+
+import graphene
+
+from ...site import models
+from ..core.mutations import BaseMutation
+from ..core.types.common import WeightUnitsEnum
+from .types import SiteSettings
+
+
+class SiteSettingsInput(graphene.InputObjectType):
+    domain = graphene.String(description='Domain name for shop')
+    header_text = graphene.String(description='Header text')
+    description = graphene.String(description='SEO description')
+    top_menu = graphene.ID(
```

Comment:
Assigning storefront menus is covered in PR #2721 - I've added dedicated mutations for that.